src/main/java/mate/academy/internetshop/controllers/IndexController.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.internetshop.controllers;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class IndexController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
```

Comment:
your line should be not longer than 100 symbols

Reply:
done